kumascript/macros/SVGElement.ejs

```diff
@@ -15,4 +15,4 @@ let term = $0;
 let page = await wiki.getPage(URL);
 
 %>
-<a href="<%- URL %>"<%- wiki.pageExists(URL) ? '' : ' class="new"' %>><code>&lt;<%= term %>&gt;</code></a>
+<a href="<%- URL %>"<%- true ? '' : ' class="new"' %>><code>&lt;<%= term %>&gt;</code></a>
```

Comment:
```suggestion
<a href="<%- URL %>"><code>&lt;<%= term %>&gt;</code></a>
```

Reply:
Got it.

Reply:
Removed the ternary conditional within the raw output tag `<%-`. Fixed!